dev_guide/builds/build_inputs.adoc

```diff
@@ -853,9 +853,9 @@ respective `destinationDir`. If you left `destinationDir` empty, then the
 secrets are placed in the working directory of the builder image.
 
 The same rule is used when a `destinationDir` is a relative path; the secrets
-are placed in the paths that are relative to the image's working directory. The
-`destinationDir` must exist or an error will occur. No directory paths are
-created during the copy process.
+are placed in the paths that are relative to the image's working directory.
+The `destinationDir` is created if it does not exist in the builder image.
+However, the parent directory of `destinationDir` must exist or an error will occur.
```

Comment:
i still find this phrasing confusing.  "destinationDir" is the name of the field, so "./foo/bar" is the destinationDir.  The parent dir is whatever "." is.

Since "foo/bar" won't be created if they don't exist (only "foo" will be created), it sounds like the behavior (for both relative and absolute paths) is:

"If the final directory in the destinationDir path does not exist, it will be created.  If any other directory in the destinationDir path does not exist, the build will fail"

right?